internal/service/repositories/repositories_test.go

```diff
@@ -182,8 +186,10 @@ func TestPatchRepository_ReplaceAll(t *testing.T) {
 					},
 				},
 			},
-			Approvers: map[string][]string{"group": {"newapprover1"}},
-			Archived:  ptr(true),
+			Approvers:           map[string][]string{"group": {"newapprover1"}},
+			Archived:            ptr(true),
+			Unmanaged:           ptr(false),
```

Comment:
woher kommt das unmanaged, bzw. wofür ist das wichtig? in der api taucht das nciht auf

Reply:
"Unmanaged" Gabs es schon vorher im yaml, siehe api/openapi-v3-spec.yaml